python/pyspark/mllib/linalg/distributed.py

```diff
@@ -151,6 +151,112 @@ def numCols(self):
         """
         return self._java_matrix_wrapper.call("numCols")
 
+    def computeSVD(self, k, computeU=False, rCond=1e-9):
+        """
+        Computes the singular value decomposition of the RowMatrix.
+
+        The given row matrix A of dimension (m X n) is decomposed into U * s * V'T where
```

Comment:
Like we talked about in 6485, probably best to trim the docstring lines down to 72 characters. :)